packages/test/test-end-to-end-tests/src/test/gc/gcStateResetInSummaries.spec.ts

```diff
@@ -1,359 +0,0 @@
-/*!
- * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
- * Licensed under the MIT License.
- */
-
-import { strict as assert } from "assert";
-
-import {
-	ITestDataObject,
-	TestDataObjectType,
-	describeCompat,
-} from "@fluid-private/test-version-utils";
-import { IContainer } from "@fluidframework/container-definitions/internal";
-import { ISummarizer } from "@fluidframework/container-runtime/internal";
-import { ISummaryTree, SummaryType } from "@fluidframework/protocol-definitions";
-import {
-	IContainerRuntimeBase,
-	channelsTreeName,
-	gcTreeKey,
-} from "@fluidframework/runtime-definitions/internal";
-import {
-	ITestContainerConfig,
-	ITestObjectProvider,
-	createSummarizer,
-	createTestConfigProvider,
-	waitForContainerConnection,
-} from "@fluidframework/test-utils/internal";
-
-import { defaultGCConfig } from "./gcTestConfigs.js";
-import { getGCStateFromSummary } from "./gcTestSummaryUtils.js";
-
-/**
- * Validates that when GC is disabled on a document that had run GC previously, the GC state is removed from summary
- * and all data stores are marked as referenced. It also tests the reverse scenario where GC is enabled on a document
- * that had GC disabled previously.
```

Comment:
Is there other code that we can/should remove as well, that was supporting this scenario?

Reply:
I tried to find all the places that was using this logic and removed it.